masonite/providers/QueueProvider.py

```diff
@@ -14,4 +15,6 @@ def register(self):
         self.app.bind('QueueConfig', queue)
 
     def boot(self, QueueConfig, QueueManager):
-        self.app.bind('Queue', QueueManager(self.app).driver(QueueConfig.DRIVER))
\ No newline at end of file
+        self.app.bind(
+            'Queue', QueueManager(self.app).driver(QueueConfig.DRIVER)
```

Comment:
let's put these on two different lines if PEP 8 allows for it. If not we can leave it.